lib/kfuzztest/parse.c

```diff
@@ -0,0 +1,74 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#include <linux/kfuzztest.h>
+
+int __kfuzztest_write_cb_common(struct file *filp, const char __user *buf,
+				size_t len, loff_t *off, void *arg,
+				size_t arg_size)
+{
+	if (len != arg_size) {
```

Comment:
https://www.kernel.org/doc/html/latest/process/coding-style.html: "Do not unnecessarily use braces where a single statement will do."